src/components/HomeMainContent.js

```diff
@@ -0,0 +1,109 @@
+import "./HomeMainContent.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+const HomeMainContent = () => {
+  const [categoryList, setCategoryList] = useState([]);
+  const [topSellingProductList, setTopSellingProductList] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const data = await axios.get("/api/categories");
+        console.log(data.data.categories);
+        setCategoryList(data.data.categories);
+      } catch (err) {
+        console.log(err);
+      }
+    })();
+    (async () => {
+      try {
+        const topSellingListData = await axios.get("/api/products");
+        console.log(topSellingListData.data.products);
+        setTopSellingProductList(
+          topSellingListData.data.products.filter(
+            (item) => item.is_topSeller === "Yes"
+          )
+        );
+      } catch (err) {
+        console.log(err);
+      }
+    })();
+  }, []);
+
+  return (
+    <main className="main-content-container">
+      <div className="hero-section">
+        <img
+          className="hero-image"
+          src="/assets/images/bobblehead-banner.jpg"
+          alt="banner"
+        />
+      </div>
+      <div className="features-card flex-row">
+        <div className="feature flex-row">
+          <i className="fa fa-truck"></i>
+          <div>
+            <h3>Free Shipping</h3>
+            <span>On all orders above ₹499</span>
+          </div>
+        </div>
+        <hr className="divider" />
+        <div className="feature flex-row">
+          <i className="fa fa-phone"></i>
+          <div>
+            <h3>Dedicated Support</h3>
+            <span>Quick response 24/7</span>
+          </div>
+        </div>
+        <hr className="divider" />
+        <div className="feature flex-row">
+          <i className="fa fa-hand-holding-dollar"></i>
+          <div>
+            <h3>Money Back Guarantee</h3>
+            <span>Worry-free shopping</span>
+          </div>
+        </div>
+      </div>
+      <div className="categories-card flex-row">
+        {categoryList.map((item) => (
+          <div className="single-category card-hw">
+            {/* <a href="#"> */}
+            <img
+              className="category-image card-hw"
+              src={item.categoryImage}
+              alt={item.categoryName}
+            />
+            {/* </a> */}
```

Comment:
remove comment

Reply:
Thank you for the suggestion, will remove it.